spec/controllers/conversations_controller_spec.rb

```diff
@@ -115,207 +123,347 @@
+          expect { post :create, @hash }.to change(Conversation, :count).by(1)
+        end
 
-    context 'with empty subject' do
-      before do
-        @hash = {
-          :format => :js,
-          :conversation => {
-            :subject => ' ',
-            :text => 'text debug'
-          },
-          :contact_ids => [alice.contacts.first.id]
-        }
-      end
+        it "creates a message" do
+          expect { post :create, @hash }.to change(Message, :count).by(1)
+        end
 
-      it 'creates a conversation' do
-        expect {
+        it "should set response with success to true and message to success message" do
           post :create, @hash
-        }.to change(Conversation, :count).by(1)
+          expect(assigns[:response][:success]).to eq(true)
+          expect(assigns[:response][:message]).to eq(I18n.t("conversations.create.sent"))
+          expect(assigns[:response][:conversation_id]).to eq(Conversation.first.id)
+        end
       end
 
-      it 'creates a message' do
-        expect {
+      context "with empty text" do
+        before do
+          @hash = {
+            format:       :js,
+            conversation: {subject: "secret stuff", text: "  "},
+            person_ids:   [alice.contacts.first.person.id]
+          }
+        end
+
+        it "does not create a conversation" do
+          count = Conversation.count
           post :create, @hash
-        }.to change(Message, :count).by(1)
-      end
+          expect(Conversation.count).to eq(count)
+        end
 
-      it 'should set response with success to true and message to success message' do
-        post :create, @hash
-        expect(assigns[:response][:success]).to eq(true)
-        expect(assigns[:response][:message]).to eq(I18n.t('conversations.create.sent'))
-        expect(assigns[:response][:conversation_id]).to eq(Conversation.first.id)
-      end
-    end
+        it "does not create a message" do
+          count = Message.count
+          post :create, @hash
+          expect(Message.count).to eq(count)
+        end
 
-    context 'with empty text' do
-      before do
-        @hash = {
-          :format => :js,
-          :conversation => {
-            :subject => 'secret stuff',
-            :text => '  '
-          },
-          :contact_ids => [alice.contacts.first.id]
-        }
+        it "should set response with success to false and message to create fail" do
+          post :create, @hash
+          expect(assigns[:response][:success]).to eq(false)
+          expect(assigns[:response][:message]).to eq(I18n.t("conversations.create.fail"))
+        end
       end
 
-      it 'does not create a conversation' do
-        count = Conversation.count
-        post :create, @hash
-        expect(Conversation.count).to eq(count)
-      end
+      context "with empty contact" do
+        before do
+          @hash = {
+            format:       :js,
+            conversation: {subject: "secret stuff", text: "text debug"},
```

Comment:
These remarks were marked as *info* by Pronto which, consequently didn't fail and didn't prevent me to push to the repo. Maybe there's a setting womewhere to fiw that?